src/components/pages/homePage/HomePage.js

```diff
@@ -0,0 +1,13 @@
+import Header from '../../header/Header';
+import SectionStart from '../../section-start/SectionStart';
+
+function HomePage() {
+  return (
+    <div className="home-page">
```

Comment:
if you use this <div> only as a wrapper, you can youse Fragments instead. <></>
https://ru.reactjs.org/docs/fragments.html